modules/solutions/activity-explorer/widgets/OpenTracingList/src/OpenTracingList.jsx

```diff
@@ -1,7 +1,6 @@
 import React, {Component} from "react";
 import Widget from "@wso2-dashboards/widget";
 import VizG from 'react-vizgrammar';
-import WidgetChannelManager from './utils/WidgetChannelManager'
```

Comment:
Shall we remove the WidgetChanelManager.js file packed in under src/utils since we are not using it?

Reply:
@Niveathika92 : done the requested changes